README.md

```diff
@@ -44,94 +100,213 @@ Open inbound ports on your security group. Use the table below as a guide. Make
-* You need to input the [AWS IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)'s _ access key ID_ and *secret_access_key*.
-* You need to have the name of the [S3 bucket you created earlier](#makebucket) handy as the install script will ask for it. The name you input has to be the exact name you gave it when you created the bucket.
-* All metadata reside in an Elasticsearch database. Make sure you have the domain name of that Elasticsearch instance handy.
-* Have the domain name of the _dos-dss server_ handy.
+Now you can begin installation. You will clone this repository on VM and
+run the bootstrap script. Be sure to set your branch to
+`feature/commons` as these instruction are specific to this branch.
+
+1. SSH onto your VM you created.
+
+1. clone this repo
+
+   ```
+   git clone https://github.com/DataBiosphere/cgp-deployment.git
+   ```
+1. Change the repo's directory
+   ```
+   cd cgp-deployment
+   ```
+
+1. Check out the correct branch
+   ```
+   git checkout feature/commons
+   ```
+
+1. Run the install script
+   ```
+   sudo bash install_bootstrap
+   ```
+
+The install script will ask for lots of information. Here we'll explain
+what to put for each step.
+
+1. First the Installer asks to install Docker and other dependencies.
+   These are necessary to continue.
 
-### Running the Installer
-Once you have collected the above information, clone the repository on VM and run the bootstrap script. Be sure to set your branch to `feature/commons` as these instruction are specific to this branch.
+1. Next, the install will ask to launch the public-facing gateway nginx
+   server. You can do this in either `dev` mode or `prod` mode. For more
+   details see the [details for installing in dev vs. prod
+   mode](#devprod).
 
-    $ git clone https://github.com/DataBiosphere/cgp-deployment.git
-    $ cd cgp-deployment
-    $ sudo bash install_bootstrap
+1. Now you will have to decide whether to launch boardwalk in `dev` or
+   `prod` modes.
 
-The `install_bootstrap` script will ask you to configure each service interactively. Specifically, you need to decide on whether you require a production (`prod` mode) or a development (`dev` mode) environment. Note that this decision can be made for _Common_ and _Boardwalk_ independently. For details regarding _Boardwalk_ see the [README](boardwalk/README.md).
+1. Next you are asked to decide if you want an authorization whitelist.
+
+   If you decide to use a whitelist you will need set it up with
+   [bouncer](https://github.com/ucsc-cgp/bouncer#how-to-use). You will
+   also be prompted for a project name and a contact email which will be
+   used in error messages for users who aren't authorized, but attempt
+   to access the data.
+
+1. You will now have to enter the hostname of dcc-dashboard. This is the
+   domain you made in Rounte 52, above. You need to know the name of the
```

Comment:
`Route 53`

Reply:
And can you make `above` be a link to the section? I was looking for it, but had a hard time finding it because the header says `Make a routing to the VM`. I entered a separate suggestion to rename the title, but even so, it would be nice to have a link as well.

Reply:
Sure that would help, I agree.